src/stravalib/model.py

```diff
@@ -38,6 +38,7 @@
 from dateutil.parser import ParserError
 from pydantic import (
     AliasChoices,
+    AnyHttpUrl,
```

Comment:
I read a bit more and this is an actual url object return. this could break people's code the way i've added it. so please let me know if this isn't ideal. if we agree to use it we can then check any field that expects a url (vs just accepting a string)